schema-loader/src/main/java/com/scalar/db/schemaloader/command/SchemaLoaderCommand.java

```diff
@@ -80,48 +82,78 @@ private static class DeleteOrRepairTables {
             "Repair tables : it repairs the table metadata of existing tables. When using Cosmos DB, it additionally repairs stored procedure attached to each table",
         defaultValue = "false")
     boolean repairTables;
+
+    @Option(
+        names = {"-A", "--alter"},
+        description =
+            "Alter tables : it will add new columns and create/delete secondary index for existing tables. "
+                + "It compares the provided table schema to the existing schema to decide which columns need to be added and which indexes need to be created or deleted",
+        defaultValue = "false")
+    boolean alterTables;
   }
 
   @Override
   public Integer call() throws Exception {
     logger.info("Config path: {}", configPath);
     logger.info("Schema path: {}", schemaFile);
 
-    if (deleteOrRepairTables == null) {
-      Map<String, String> options = new HashMap<>();
-      if (replicationStrategy != null) {
-        options.put(CassandraAdmin.REPLICATION_STRATEGY, replicationStrategy.toString());
-      }
-      if (compactionStrategy != null) {
-        options.put(CassandraAdmin.COMPACTION_STRATEGY, compactionStrategy.toString());
-      }
-      if (replicaFactor != null) {
-        options.put(CassandraAdmin.REPLICATION_FACTOR, replicaFactor);
-      }
-      if (ru != null) {
-        options.put(DynamoAdmin.REQUEST_UNIT, ru);
-      }
-      if (noScaling != null) {
-        options.put(DynamoAdmin.NO_SCALING, noScaling.toString());
-      }
-      if (noBackup != null) {
-        options.put(DynamoAdmin.NO_BACKUP, noBackup.toString());
-      }
-
-      SchemaLoader.load(configPath, schemaFile, options, coordinator);
-    } else if (deleteOrRepairTables.deleteTables) {
+    if (mode == null) {
+      createTables();
+    } else if (mode.deleteTables) {
       SchemaLoader.unload(configPath, schemaFile, coordinator);
+    } else if (mode.repairTables) {
+      repairTables();
     } else {
```

Comment:
Very minor suggestion:
```suggestion
    } else if (mode.alterTables) {
```

I feel it makes the code slightly clearer (, but actually either is fine).